sample_local_testing_program.py

```diff
@@ -187,15 +189,16 @@ def main(test=False):
 
         # Step 2
         print()
-        print ("Step 2: Create types, streams, and data")
+        print("Step 2: Create types, streams, and data")
         if needData:
             createData(ocsClient)
 
         # Step 3
         print()
         print("Step 3: Create a data view")
-        dataView = DataView(id=sampleDataViewId,name=sampleDataViewName,description=sampleDataViewDescription)
-        dataViews = ocsClient.DataViews.postDataView(namespaceId, dataView)
+        dataView = DataView(id=sampleDataViewId, name=sampleDataViewName,
+                            description=sampleDataViewDescription)
+        dataView = ocsClient.DataViews.postDataView(namespaceId, dataView)
 
         # Step 4
         print()
```

Comment:
My comment is on the wrong line, but line 213 -- dv.Queries.append is broken, because we now default Queries to None in stead of an empty array.  I saw this in the code, but was wanting to check it my own code, but now I see it in the sample-local testing app.

Reply:
I have it fixed in my local copy of the data views sample, along with other fixes for multiple samples once this PR is merged.